docs/source/upcoming_release_notes/1160-Update_lxt_limits.rst

```diff
@@ -0,0 +1,32 @@
+1160 Update lxt limits
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- N/A
+
+Device Updates
+--------------
+- LaserTiming
+- LaserTimingCompensation
+- LxtTtcExample
```

Comment:
```suggestion
- Updates limits for LaserTiming, LaserTimingCompensation, and LxtTtcExample from +/-10us to +/-100us
```

Maybe something like this?  I'd combine your comment into one line to make sure the change info is kept together

Reply:
I think the release notes here are the only adjustments needed to the PR imo